tables/zenodo-upload/README.md

```diff
@@ -0,0 +1,88 @@
+|   2C     |    Onocoprint summarizing high-grade glioma alterations across samples for top 20 mutated genes         |
+|   2D     |    Onocoprint summarizing other CNS tumor alterations across samples for top 20 mutated genes         |
+|   3A     |    Barplot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types         |
+|   3B     |  Heatmap of co-occurrence and mutual exclusivity of nonsynonymous mutations between genes           |
+|   3C     |  Scatterplot showing correlation of SV and CNV breaks            |
+|   3D     |  Barplot showing chromothripsis frequency for all cancer groups with N >= 3 tumors           |
+|   3E     |  Sina plots of RefSig signature weights across cancer groups         |
+|   4A     |  ROC curve for TP53 classifier         |
+|   4B     |  Violin/strip plots of TP53 scores across TP53 alteration status           |
+|   4C     |   Violin/strip plots of TP53 expression [log(FPKM)] across TP53 alteration status            |
+|   4D     |  Box/strip plots of TP53 scores and telomerase scores across cancer groups          |
+|   4E    |    Heatmap of RegSg signatures in patients with a hypermutator phenotype       |
+|   4F     |  Forest plot from survival analysis exploring TP53 and telomerase score effects          |
+|   4G     |  Forest plot from survival analysis exploring HGG molecular subtype effects          |
+|   4H     |  Kaplan-Meier cure of HGG tumors by molecular subtype           |
+|   5A     |  UMAP of tumors colored by broad histology           |
+|   5B     |  Heatmap of GSVA scores across cancer groups           |
+|   5C     |  Box/strip plots of quanTIseq scores across cancer groups           |
+|   5D     |  Forest plot from survival analysis exploring CD274 expression and immune cell proportion effects           |
+|   5E     |  Box/strip plot of CD274 expression across medulloblastoma subtypes          |
+|   S2A     |             |
+|   S2B     |             |
+|   S2C     |             |
+|   S3D     |             |
+|   S2E     |             |
+|   S2F     |             |
+|   S2G     |             |
+|   S2H     |             |
+|   S2I     |             |
+|   S3A     |             |
+|   S3B     |             |
+|   S3C     |             |
+|   S3D     |             |
+|   S3E     |             |
+|   S4A     |             |
+|   S4B     |             |
+|   S5A     |             |
+|   S5B     |             |
+|   S5C     |             |
+|   S6A     |             |
+|   S6B     |             |
+|   S6C     |             |
+|   SD6     |             |
+|   S6E     |             |
+|   S6F     |             |
+|   S7A     |             |
+|   S7B     |             |
+|   S7C     |             |
+|   S7D     |             |
+|   S7E     |             |
+|   S7F     |             |
+|   S7G     |             |
+|   S7H     |             |
+|   S7I     |             |
+
+
+
+
+
+
+|--------|--------|------------------|
```

Comment:
I assume you want a header row here, and probably a bit of a description of the table? (Maybe in the next PR, but it seems like it would make sense to add here)

Reply:
Hm what I actually meant was not to have this here at all (this is is just a "typo") but now I'm very curious what you _thought_ this table would have been so I can maybe add it!